### PR TITLE
Fix the whitepaper form

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Now that formValidation is built into the main package with is loaded on all pages. I have renamed the functions to not clash with the global functions.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to `/whitepapers/robotics` and fill in the form and see it works
- Go to `/whitepapers/beta/robotics` and fill in the form and see it works

### CODE REVIEW

**Comment by caldav** [comment]
@anthonydillon any way to keep the `mktoForm_<id>` format? Tag manager consumes it and sends it to GA as event data, allowing us to identify specific form submissions.

File: templates/whitepapers/beta/shared/form.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!-- MARKETO FORM -->
-<script src="//assets.ubuntu.com/v1/37b1db88-jquery.min.js"></script>
+<!-- <script src="//assets.ubuntu.com/v1/37b1db88-jquery.min.js"></script>
```
Comment (steverydz) [inline templates/whitepapers/beta/shared/form.html]: These should be removed if not needed

**Comment by anthonydillon** [comment]
Removed the submit code from the whitepapers and reused the formValidation code to submit with a callback to reveal the content.

**Comment by anthonydillon** [comment]
@caldav could you have another look please

**Comment by caldav** [comment]
:+1: thanks!

**Pull request merged**